spec.bs

```diff
@@ -1424,6 +1425,23 @@ Issue: This would ideally be replaced by a more descriptive algorithm in Infra.
 </div>
 
 <div algorithm>
+To <dfn>round a value</dfn> given a {{double}} |value|:
+  1. If |value| is not a [=valid floating-point number=], return |value|.
+  1. Let |valueExp| be |value|'s IEEE 754 biased exponent field minus 1023.
+  1. Let |normValue| be |value| multiplied by 2<sup>(&minus;1 &times; |valueExp|)</sup>.
+  1. If |valueExp| is less than &minus;128:
+    1. If |value| is less than 0, return &minus;0.
+    1. Otherwise, return 0.
+  1. If |valueExp| is greater than 127:
+    1. If |value| is less than 0, return &minus;&infin;.
+    1. Otherwise, return &infin;.
+  1. Let |precisionScaledValue| be |normValue| multiplied by 256.

```

Comment:
Don't we use different values than 256 in different parts of FLEDGE? In the implementation [0], this is a parameter.

[0] https://source.chromium.org/chromium/chromium/src/+/main:content/browser/interest_group/interest_group_auction_reporter.cc;l=282;drc=9d5a24607b3a18c67a8672524630ddf2688f86a3

Reply:
From talking with Paul offline, it sounds like this comment is obsolete and can be resolved.

Reply:
Yes -- for posterity, the long-term state we want is for all of these to eventually be 256 as well, so a parameter is not needed.